app/shared/layouts/navigation/RightSide.js

```diff
@@ -14,6 +14,7 @@ import Button from '@shared/buttons/Button';
 import { USER_ROLES } from 'helpers/userHelper';
 
 export default function RightSide({ campaignStatus }) {
+  // TODO: campaignRequestPending not releveant anymore?
```

Comment:
Def not.. we'll need to rip out all the logic related to the Campaign Managers/Team Members feature since we didn't implement any of that business logic in the new API

Reply:
I went ahead and created a task for this here: https://goodparty.atlassian.net/browse/WEB-3583